deploy/helm/charts/templates/_helpers.tpl

```diff
@@ -59,9 +59,7 @@ Define meta labels for openebs lvm-localpv components
 */}}
 {{- define "lvmlocalpv.common.metaLabels" -}}
 chart: {{ template "lvmlocalpv.chart" . }}
-heritage: {{ .Release.Service }}
 openebs.io/version: {{ .Chart.AppVersion | quote }}
-role: {{ .Values.role | quote }}
```

Comment:
same comment on the role as the one on zfs

Reply:
Reverted this.